source/App/libvvcdec/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# executable
+set( DLL_NAME libvvcdec )
```

Comment:
Would this not produce `liblibvvcdec.so` etc? Is this desirable?

Reply:
True. I changed it to vvcdec. This will make a `libvvdec.so` or `libvvdec.dylib` on linux/mac and a vvdec.dll on windows.